### PR TITLE
Loadmill dashboards

### PR DESCRIPTION
**Pull request by yigaldviri**
# Summary

This PR is meant to add a dashboard of 6 widgets to the Quickstarts. 
Only a dashboard is added. Alerts aren't. 

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/loadmill/loadmill.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
+    "name": "test",
```
Comment (brammerl) [inline dashboards/loadmill/loadmill.json:2]: Let's rename this to something more meaningful

File: dashboards/loadmill/loadmill.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    "description": null,
+    "pages": [
+      {
+        "name": "test",
```
Comment (brammerl) [inline dashboards/loadmill/loadmill.json:6]: Same here

File: quickstarts/loadmill/config.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Loadmill L.T.D
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
```
Comment (brammerl) [inline quickstarts/loadmill/config.yml:23]: Hello! If you want your quickstart to appear under a specific category please use the keywords here in the query. They map to specific categories. 

https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

Comment (yigaldviri) [reply quickstarts/loadmill/config.yml:23]: I added `network` though its not the best category. Is there an option to add a testing category?

**Comment by yigaldviri** [comment]
Thanks for the feedback. I made some changes accordingly

File: quickstarts/loadmill/config.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  - network
+
+# Reference to dashboards to be included in this quickstart
+dashboards:
+  - loadmill
+
+# Documentation references
+documentation:
+  - name: Loadmill Docs
+    url: docs.loadmill.com
```
Comment (zstix) [inline quickstarts/loadmill/config.yml:38]: Our validation expects the URl to be include the protocol (`https://`):
```suggestion
    url: https://docs.loadmill.com
```

**Review by zstix — CHANGES_REQUESTED** [review]
One small adjustment needed. Also, you branch is out of date with the `main`, could you place pull the latest into your branch? Thanks!!

**Comment by yigaldviri** [comment]
@zstix - done

**Comment by yigaldviri** [comment]
@zstix - done

**Review by zstix — APPROVED** [review]

**Pull request merged**